app/models/card.rb

```diff
@@ -4,7 +4,7 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
-  scope :pending, -> { where('review_date >= ?', Time.now).order('RANDOM()') }
+  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.